extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/configuration_validation/ClassWithConfigProperties.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkus.hibernate.validator.runtime.configuration_validation;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.enterprise.inject.spi.CDI;
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import javax.validation.Validator;
+
+/**
+ * Holds the class and the configuration properties i.e class fields annotated with
+ * {@link org.eclipse.microprofile.config.inject.ConfigProperty}
+ */
+public class ClassWithConfigProperties {
+    private final Class<?> aClass;
+    private final List<String> configurationProperties;
+    static List<ClassWithConfigProperties> classWithConfigurationProperties;
```

Comment:
I personally find the name of the field confusing. I would have named it `instances` or something

Reply:
Thanks, I like the proposed naming.